.github/workflows/test-accessibility.yml

```diff
@@ -0,0 +1,26 @@
+name: Accessibility Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
if you want to run this on each PR it makes more sense to test against a local version of the app (Michael got this set up). otherwise you can run against AWS URLs but makes more sense to do this on a cron every night

Reply:
that makes more sense! will rework to use local version and possibly set up a separate scheduled one to run against AWS endpoints?